### ISSUE TITLE
How to turn off snipes and all captures? (Only want to farm pokeballs)

### ISSUE THREAD

**Issue by Victh** [NECROBOTIO/NecroBot#1673]

**Comment by Newt0570** [comment]
I'm pretty sure snipes are off by default (there's a setting "snipe at pokestops" or something to change to false), and you can add every pokemon to the "pokemon to ignore" list. It should also be configured by default to throw away the worst balls when your inventory fills up.

**Comment by Cuuup** [comment]
Set  "UsePokemonToNotCatchFilter": true,
and  "SnipeAtPokestops": true,

It won't cancel all captures but, for the most part but necrobot will avoid all other pokemons that you put in "PokemonsToIgnore": [

**Comment by Victh** [comment]
Let me rephrases my question: Is there any way to make the bot only sniping '_rares_' (the ones provided by PogoLocationFeeder) Pokemon only?

Sometimes when I'm low on balls, I just want to skip weaker pokemon. Adding every other monster might work, but that's took a lot more effort for something seems so simple. 

I might want to wait for a more efficient solution before using the answers above. But really thanks for the responses!

**Comment by Andrerm124** [comment]
I've submitted a pull request to add an option to ignore all pokemon NOT on the sniper list. I think this solves your issue, thanks for the idea.

**Comment by recon88** [comment]
#1874
